### PR TITLE
Add 80ms delay between each component sync

### PR DESCRIPTION
**Pull request by keichinger**
This prevents flooding the Management API and preventing the HTTP 429 error

### CODE REVIEW

File: src/Manager/Sync/ComponentSync.php [hunk 1]
```diff
@@ -67,6 +67,8 @@ private function syncComponents (TorrStyle $io) : void
 			$io->write("Syncing {$key} ");
 			$performedAction = $this->managementApi->syncComponent($config->config, $config->groupLabel);
 			$io->writeln(\sprintf("%s <fg=green>✓</>", $performedAction->value));
+
+			\usleep(80000);
```
Comment (apfelbox) [inline src/Manager/Sync/ComponentSync.php]: I would rather implement a rate limiter here instead.

**Comment by apfelbox** [comment]
I updated the implementation to use a rate limiter instead

**Review by apfelbox — APPROVED** [review]

**Comment by apfelbox** [comment]
@keichinger please review the latest changes and merge if they look ok

**Comment by keichinger** [comment]
@apfelbox looks good!

**Pull request merged**